pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -32,6 +32,17 @@ import (
 
 const (
 	sourceVMInitialState = "vmimport.v2v.kubevirt.io/source-vm-initial-state"
+	// AnnCurrentProgress is annotations storing current progress of the vm import
+	AnnCurrentProgress = "vmimport.v2v.kubevirt.io/progress"
+	// constants
+	progressStart            = "0"
+	progressValidation       = "10"
+	progressUpdatingSourceVM = "20"
+	progressCreatingVM       = "30"
+	progressCopyingDisks     = "40"
+	progressStartVM          = "90"
+	progressDone             = "100"
+	progressForCopyDisk      = 40
```

Comment:
I see here `progressForCopyDisk` and `progressCopyingDisks`. One with int and the other with string as value. Do we need both?

Reply:
`progressForCopyDisk` is used to calculate the progress from the number of disks as:
`progressCopyingDisks + (progressForCopyDisk/numberofdisksDone)`

Reply:
So when all disks are done we are at 80% and when we start copying disk we are at 40%. When there are two disks, and first is done we are at 60%. When both we are at 80%